engines/common/nucleus/particles/menu.html.twig

```diff
@@ -44,6 +44,13 @@
         {% set target = item.target != '_self' ? ' target="' ~ item.target|e ~ '"' %}
         {% set rel = item.rel ? ' rel="' ~ item.rel|e ~ '"' %}
 
+        {% set visibleChilds = false %}
+        {% for child in item.children %}
+            {% if child.visible and child.enabled and enabled|default(1) %}
+                {% set visibleChilds = true %}
```

Comment:
Should we break the loop?
Because if one child is visible we can skip the rest of the children.

I don't know if this is worth the effort?
Because then this TokenParser needs to be implemented... https://stackoverflow.com/a/40949346/6370595

Reply:
It doesn't matter if there are just a few children, but if the menu is huge, there will be some performance improvements on this. Generally, menus aren't that large, though...